pkgs/by-name/pe/petsc/package.nix

```diff
@@ -174,10 +174,18 @@ stdenv.mkDerivation rec {
   # the library is installed and available.
   doInstallCheck = true;
   installCheckTarget = "check_install";
-  nativeInstallCheckInputs = [ mpiCheckPhaseHook ];
+  nativeInstallCheckInputs =
+    [
+      mpiCheckPhaseHook
+    ]
+    ++ lib.optionals pythonSupport [
+      python3Packages.pythonImportsCheckHook
+    ];
+
+  pythonImportsCheck = [ "petsc4py" ];
```

Comment:
Nit: Technically, these could be separated to a different commit, as they are not part of the argument rename.